src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -281,7 +298,12 @@ protected function renderForm(string $view, array $parameters = [], Response $re
     {
         trigger_deprecation('symfony/framework-bundle', '6.2', 'The "%s::renderForm()" method is deprecated, use "render()" instead.', get_debug_type($this));
 
-        return $this->render($view, $parameters, $response);
+        $oldValue = $this->renderFormViews;
+        $this->renderFormViews = true;
+        $response = $this->render($view, $parameters, $response);
+        $this->renderFormViews = $oldValue;
```

Comment:
this should be done in a `finally` block, so that the state is always properly reset

Reply:
Good call, done! Thanks!